src/styles/star.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  text-decoration: none;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+@for $starsStatus from 1 through $active-stars {
```

Comment:
Why you use $active-stars?
In this case modifier will be work only to 4 stars. 
You need loop from 1 to 5.